get-started-microservices-on-kubernetes/simple-microservice-jms/README.adoc

```diff
@@ -0,0 +1,90 @@
+
+= getting-started
+
+The `getting-started` project is a simple Jakarta EE application with a HTTP endpoint that is running in
+https://wildfly.org[WildFly].
+
+The `src/main` folder contains a simple 'Hello world' style Jakarta EE application using JAX-RS.
```

Comment:
I haven't looked carefully at the initial part of these README pages, but it seems like they are all the same which seems wrong since the apps differ.

Reply:
I have differentiated them